packages/scandipwa/src/component/ProductActions/ProductActions.style.scss

```diff
@@ -53,13 +53,16 @@
 
     &-ShortDescription {
         font-size: 14px;
-        line-height: 20px;
         margin-block: 16px;
 
         div[itemprop="description"] li:before{
             position: static;
         }
-    }
+
+        * :not(h1):not(h2):not(h3):not(h4):not(h5):not(h6) {
```

Comment:
Hello !
Try maybe list of selectors could work? Example
:not(h1, h2, h3)
at least it looks odd.

Any other solution can be applied here ?

Reply:
I mean maybe there is list of selectors 3-4 ,where it should be placed ?

Reply:
I did try everything. I tried like this :not(h1, h2, h3) too. And I searched if there was a selector for <H> tags but it seems like there is not. So I needed to select them one by one to exclude them. Also, I searched and: not(h1, h2) is not correct syntax and it does not work. I agree it looks odd xd.  

And I have to use " * " because there are a lot of tags in use and I need to select all of them. Also not sure why it didn't work on span and div, Gonna check it out

Reply:
Well you can use mixing in this case.

Reply:
Will try, ty